picard/ui/item.py

```diff
@@ -99,6 +100,28 @@ def error_append(self, msg):
     def clear_errors(self):
         self._errors = []
 
+    def _cover_art_description(self, images, detailed=True):
```

Comment:
It would be better to define 2 functions, instead of passing boolean argument `detailed`, which is `True` only once in current code. Common code is rather small in both cases, and it would be clearer.
Also since those methods are used outside their module, I would get rid of `_` prefix:

```python
def cover_art_description(self, images):
    return str(len(images))

def cover_art_description_detailed(self, images):
    number_of_images = len(images)
    if getattr(self, 'has_common_images', True):
       ...
```